### PR TITLE
Update RD.XML for TypeConverterAttribute

### PR DESCRIPTION
**Pull request by MichalStrehovsky**
`GetDefaultAttribute` will look for a field named `Default` on things and there are code paths through which this is reachable with the `TypeConverterAttribute`.

https://github.com/dotnet/corefx/blob/b996738f877d17b0fe426c75b01f0a8ecdc21e07/src/System.ComponentModel.TypeConverter/src/System/ComponentModel/AttributeCollection.cs#L267-L269

### CODE REVIEW

**Comment by MichalStrehovsky** [comment]
@morganbr please take a look. This complements the Project N fix I sent a review for right now.

**Comment by MichalStrehovsky** [comment]
@dotnet-bot test Windows x64 Debug Build

File: src/System.ComponentModel.TypeConverter/src/Resources/System.ComponentModel.TypeConverter.rd.xml [hunk 1]
```diff
@@ -13,6 +13,9 @@
         </Type>
         <Type Name="TypeConverterAttribute">
           <AttributeImplies Activate="Required Public" />
+          
+          <!-- AttributeCollection.GetDefaultAttribute will end up reflecting on the Default field -->
```
Comment (morganbr) [inline src/System.ComponentModel.TypeConverter/src/Resources/System.ComponentModel.TypeConverter.rd.xml]: This is OK, but should we have an intrinsic or something else that should pick this up?

File: src/System.ComponentModel.TypeConverter/src/Resources/System.ComponentModel.TypeConverter.rd.xml [hunk 2]
```diff
@@ -13,6 +13,9 @@
...
+          <Field Name="Default" Dynamic="Required" />
```
Comment (morganbr) [inline src/System.ComponentModel.TypeConverter/src/Resources/System.ComponentModel.TypeConverter.rd.xml]: This makes this attribute self-rooting. Does it need to be rooted?

Comment (MichalStrehovsky) [reply src/System.ComponentModel.TypeConverter/src/Resources/System.ComponentModel.TypeConverter.rd.xml]: The thing I would want to go for is "if the owning type is getting any sort of metadata, also root and generate metadata for this field", but I don't believe we have means to express that in RD.XML.

Rooting this field is pretty harmless.

**Pull request merged**